system_tests/logging_.py

```diff
@@ -205,10 +186,12 @@ def test_list_metrics(self):
                          set([DEFAULT_METRIC_NAME]))
 
     def test_reload_metric(self):
+        from gcloud.exceptions import Conflict
+        retry = RetryErrors(Conflict)
         metric = Config.CLIENT.metric(
             DEFAULT_METRIC_NAME, DEFAULT_FILTER, DEFAULT_DESCRIPTION)
         self.assertFalse(metric.exists())
-        metric.create()
+        retry(metric.create)()
```

Comment:
Speculating, but write operations may hit a different tier of servers than reads?  At any rate, I saw the failure in the wild.